app/Http/Controllers/ApplicationController.php

```diff
@@ -146,9 +145,16 @@ public function edit($id)
       $hear_about = Scholarship::getCurrentScholarship()->hear_about_options;
       $choices = Application::formatChoices($hear_about);
 
+      // We have to pass uploads to the view, so set null if there aren't any
+      if ($application['upload']) {
```

Comment:
We could wrap it in an empty check, but if it's empty then `$application['upload']` is also false so not sure if that's necessary or what the standard protocol on that is.

`$application` is actually an array here because of how `Application::getUserApplication($id)` works, but we should probably change that. I'd like to break that out into a separate PR though if possible 😄